README.md

```diff
@@ -369,7 +369,8 @@ class LitAutoEncoder(pl.LightningModule):
 
 The lightning community is maintained by
 - [10+ core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, and Ph.D. students from top AI labs.
-- 400+ community contributors.
+- 480+ community active contributors.
+- read how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)
```

Comment:
Why not add the content directly in the docs instead?

Reply:
then we can add it on both places, but many people reds just readme